src/app/complex/complex-results/complex-results.component.ts

```diff
@@ -153,22 +192,31 @@ export class ComplexResultsComponent implements OnInit, AfterViewInit {
 
   set complexSearch(value: ComplexSearchResult) {
     this._complexSearch = value;
+    this.setFirstDisplayType(); // if only one complex is in the search result, the display is list
```

Comment:
This comment is outdated. If only one complex, then we redirect to the detailed complex page